src/kibana/plugins/dashboard/components/modal/searchAuditor.js

```diff
@@ -0,0 +1,65 @@
+define(function (require) {
+    var app = require('modules').get('app/dashboard');
+    app.factory('searchAuditor', function (elasticSearchFields,
+                                           Restangular) {
+        var previousQuery = {
+            query    : "",
+            fromDate : "",
+            toDate   : ""
+        };
+        var formatQuery = function(query) {
+            var _query = _.isArray(query) ? query : [query];
+            if (_.isArray(_query)) {
+                var count = 0;
+                _.each(_query, function($q) {
+                    $q = elasticSearchFields.convertQuery($q);
+                    _query[count] = $q;
+                    ++count;
+                });
+
+            } else {
+                _query = elasticSearchFields.convertQuery(_query);
+            }
+            return _query;
+        };
+        return {
+            autoCapQuery : function(query) {
+                return formatQuery(query)[0];
+            },
+            logQuery : function(query, from, to) {
+                if (!from || !to || 
+                    query === "" || 
+                    query === "*" || 
+                    query === "+Captured:true") {
```

Comment:
@yannick-polius please add a comment here stating that these are  here so that they are removed from the log auditing.